CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 ## Below is the original changelog of the tobrun/flutter-mapbox-gl project, before the fork.
 
+## 0.12.0, April 12, 2020
```

Comment:
I'd like to move this section of changelogs above the "## Below is the original changelog..." heading and put it under a separate heading like:
 "## Changes were cherry-picked/ported from the following releases of the tobrun/flutter-mapbox-gl project into this project"